tests/datalab/test_cleanvision_integration.py

```diff
@@ -0,0 +1,200 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+import pandas as pd
+
```

Comment:
wherever you are creating the `image_dataset` out of the files like `tests/datalab/data/random_class_1/image_604.png` etc. Please move those files out of this repo, download them from internet, and then construct the `image_dataset`. This is worthwhile to keep the repo size limited (we hardly ever want to store any data inside this repo)

Reply:
moved out to assets repo